scripts/install.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+set -e
+
+OCDEV_VERSION="latest"
```

Comment:
this should have commend what it is and what values are possible and

Reply:
fixed